src/oauth.test.ts

```diff
@@ -65,9 +78,504 @@ describe.sequential("OAuth", () => {
       "client_secret_basic",
     ]);
 
-    expect(metadata.scopes_supported).to;
-
```

Comment:
This would've totally worked, I swear. I have no idea how this didn't error in #154